src/com/google/enterprise/adaptor/fs/FsAdaptor.java

```diff
@@ -1177,14 +1171,13 @@ public DocId apply(Path path) {
                   }
                 }));
       } catch (IOException | WrappedException | InterruptedException e) {
-        log.log(Level.WARNING, "Failed to push contents of directory " + dir,
-                e);
+        log.log(Level.WARNING, "Failed to push child DocIds of " + source, e);
       } finally {
         try {
-          setLastAccessTime(dir, lastAccessTime);
+          setLastAccessTime(source, lastAccessTime);
         } catch (IOException e) {
-          log.log(Level.WARNING, "Failed restore last access time for " + dir,
-                  e);
+          log.log(Level.WARNING, "Failed restore last access time for "
+                   + source, e);
```

Comment:
Strange indentation. Maybe break before the whole string. Also, maybe s/Failed/Failed to/, since we're editing this line?